.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- [ ] Consider removing this `test.md` from the `.gitignore` file as you do not have it in your current project directories 😄